### ISSUE TITLE
Blinds work, Terrace (sun) screens are not showing up

### ISSUE THREAD

**Issue by TB404** [nonolk/domoticz_tahoma_blind#8]
All the blinds added to the Conexoon (visible in the Somfy Windows app) are showing up and working great.

A Terrace screen added to the Conexoon, only visible in the Somfy Terrace app) does not show up at all. Not sure if that is by design or a bug. 
I would love to include the Terrace screen, SUNEA io as default name,  in my domoticz set up.

**Comment by nonolk** [comment]
Hello TB404,

It’s not a bug, it’s just because your device is from a plugin point of view of an unknown type.
I don’t know if I will be able to support it, but a good starting point is to provide some information about your device by activating the debug log in the plugin, and by restarting it/domoticz. 

Please also provide information about plugin and domoticz version.

Best regards,

**Comment by TB404** [comment]
Thanks for your answer, Somfy may see them as different devices, but as they use the same local remote controls (without the conexoon) I am pretty sure that the controls cant be that different really.
Here are the logs
https://pastebin.com/jpXheggQ
(Enabled debugging and clicked Update.)

Plugin version:
2020-05-17 20:05:49.754 Status: (Conexoon) Initialized version 1.0.3, author 'nonolk'
2020-05-17 20:05:49.762 Status: (Conexoon) Starting Tahoma blind plugin

Domoticz version:
Version: 2020.2
Build Hash: b63341bc0
Compile Date: 2020-04-26 13:47:55
dzVents Version: 3.0.2
Python Version: 3.7.3 (default, Dec 20 2019, 18:57:59) [GCC 8.3.0]

Happy to help in any way, I am not too technical on the coding side

**Comment by TB404** [comment]
Actually, I just noticed you have version 1.0.5 available. I replaced the files in the plugin and enabled the plugin again. It now found the other device and it is working. My sincere apologies for not paying attention, not running the latest version and I really appreciate the plugin! It is awesome and I am such a happy user!

Although I've updated the files, domoticz still says the version is 1.0.3 but it does work now.
